README.md

```diff
@@ -6,7 +6,7 @@
 
 ## Description
 
-This repository is part of the [Xperience by Kentico Migration Tool](https://github.com/Kentico/xperience-by-kentico-migration-tool).
+This repository is part of the [Xperience by Kentico Migration Tool](https://github.com/Kentico/xperience-by-kentico-kentico-migration-tool).
```

Comment:
```suggestion
This repository is part of the [Xperience by Kentico Migration Toolkit](https://github.com/Kentico/xperience-by-kentico-migration-toolkit).
```

Good catch! Both the URL _and_ link text need updated.

I think the change from `Toolkit` to `Tool` happened during a recent renaming. The [Xperience by Kentico Migration Toolkit](https://github.com/Kentico/xperience-by-kentico-migration-toolkit) (the Toolkit includes all migration tools) is what we want to link to here.

Reply:
```suggestion
This repository is part of the [Xperience by Kentico Migration Toolkit](https://github.com/Kentico/xperience-by-kentico-migration-toolkit).
```